### PR TITLE
Display stream information for twitch users

### PR DESCRIPTION
**Pull request by M4tthewDE**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

Display stream information if twitch user is live.

### CODE REVIEW

File: internal/resolvers/twitch/user_loader.go [hunk 1]
```diff
@@ -48,17 +59,36 @@ func (l *UserLoader) Load(ctx context.Context, login string, r *http.Request) (*
 		return nil, cache.NoSpecialDur, resolver.ErrDontHandle
 	}
 
+	streamResponse, err := l.helixAPI.GetStreams(&helix.StreamsParams{UserLogins: []string{login}})
+	if err != nil {
```
Comment (pajlada) [inline internal/resolvers/twitch/user_loader.go:63]: Let's handle `err != nil` and `len(streamResponse.Data.Streams) != 1` in the same way: Output it as a userResponse instead

File: pkg/config/config.go [hunk 2]
```diff
@@ -66,7 +66,7 @@ func init() {
 	pflag.Uint64("max-content-length", 5*1024*1024, "Max content size in bytes - requests with body bigger than this value will be skipped")
 	pflag.Bool("enable-animated-thumbnails", true, "When enabled, will attempt to use libvips library to build animated thumbnails. Can increase CPU usage and cache storage by a lot. Enabled by default")
 	pflag.Uint("max-thumbnail-size", 300, "Maximum width/height pixel size count of the thumbnails sent to the clients.")
-	pflag.Duration("twitch-username-cache-duration", 1*time.Hour, "Cache timeout for twitch usernames")
+	pflag.Duration("twitch-username-cache-duration", 10*time.Minute, "Cache timeout for twitch usernames")
```
Comment (pajlada) [inline pkg/config/config.go:69]: Document this in the readme

Comment (M4tthewDE) [reply pkg/config/config.go:69]: I changed the deefault value in `config.yaml`, not sure where to place it in the README

Comment (pajlada) [reply pkg/config/config.go:69]: oops I meant the changelog - added in [`3f4730c` (#714)](https://github.com/Chatterino/api/pull/714/commits/3f4730c97399b2c2ad6836350e17da1d95ed4513)

**Review by pajlada — APPROVED** [review]

**Pull request merged**